### DISCUSSION TITLE
How do I request page 2 of the API? (pagination)

### DISCUSSION THREAD

**Discussion by jackinstagamez** [pterodactyl/panel#3095] [Support]
Hi guys,

I am developing a piece of software that contacts your API and I can't find out how to request the next page of the API. I also can't see it in the documentation anywhere.

Any help would be greatly appreciated!

Jack

**Comment by schrej** [comment]
You can control paging with the `per_page` and `page` query parameters. For the second page simply append `?page=2`. The `meta` object in the response contains how many pages exist and so on.
